docs/StandardLibraryProgrammersManual.md

```diff
@@ -22,24 +24,24 @@ We use two spaces as the unit of indentation. We don't use tabs.
 
 To break long lines, please closely follow the indentation conventions you see in the existing codebase. (FIXME: Describe in detail.)
 
-Our primary rule is that if we need to break a list (such as arguments, tuple or array/dictionary literals, generic type parameters, etc.), then we always put each item on its own line, indented by +1 unit, even if a series of items would fit on a single line together. 
+Our primary rule is that if we need to put a line break anywhere in the middle of a list (such as arguments, tuple or array/dictionary literals, generic type parameters, etc.), then we need to go the full way, and put each item on its own line, indented by +1 unit, even if some of the items would fit on a single line together.
```

Comment:
Nit:
```suggestion
Our primary rule is that if we need to put a line break anywhere in the middle of a list (such as arguments, tuple or array/dictionary literals, generic type parameters, etc.), then we should go all the way, and put each item on its own line, indented by +1 unit, even if some of the items would fit on a single line together.
```

Reply:
Another one:
```suggestion
Our primary rule is that if we need to insert a line break anywhere in the middle of a list (such as arguments, tuple or array/dictionary literals, generic type parameters, etc.), then we need to go the full way, and put each item on its own line, indented by +1 unit, even if some of the items would fit on a single line together.
```

Reply:
I sneakily edited your first suggestion to change "then we need to go" to "then we must go" (RFC 2119 style), to emphasize that this is not something people can opt out of if they don't like it. 🤓